docs/user-guide/mvd-3270.md

```diff
@@ -0,0 +1,120 @@
+# 3270 Terminal
+The 3270 Terminal Display Emulator plugin provides a user interface that emulates the basic functions of IBM 3270 family terminals. On the "back end," the plugin and the Zowe Application Server connect to any standard TN3270/E server.
```

Comment:
Two things:
- Our style guide [word usage](https://docs.zowe.org/stable/contribute/contributing#word-usage) standard is to use **plug-in** (instead of plugin)
- No need to use quotes w/ **back end**.